CHANGELOG.rst

```diff
@@ -0,0 +1,100 @@
+==========
+Changelog
+==========
+
+This project adheres to `Semantic Versioning <https://semver.org/>`_.
+
+
```

Comment:
Should the new Release notes also be included here? What's the reason for including 2 different changelogs?

Reply:
Good point. I think the .rst is the latest version we use and I tried to transfer all changes from the other log there. Shall we delete CHANGELOG.md?